spec/index.bs

```diff
@@ -1334,14 +1325,12 @@ For the {{Element/insertAdjacentHTML()}} method, execute the
 [$Enforce a Trusted Type algorithm$] with input argument set to the *text*
 parameter value.
 
-Issue: do we need to specify that type argument should be {{TrustedHTML}}
-twice herein.
-
```

Comment:
Not really, this is implied from the sink definition (it accepts `HTMLString`, which is then converted in the algorithm to expectedType `TrustedHTML`)